src/java/org/apache/cassandra/service/paxos/CommitVerbHandler.java

```diff
@@ -50,6 +52,9 @@ public void doVerb(Message<Commit> message)
         Tracing.trace("Enqueuing acknowledge to {}", message.from());
         Message.Builder<NoPayload> reply = message.emptyResponseBuilder();
         SensorsCustomParams.addWriteSensorToResponse(reply, sensors, context);
+
+        // no need to calculate outbound internode bytes because the response is NoPayload
+        SensorsCustomParams.addInternodeBytesSensorToResponse(reply, sensors, context);
```

Comment:
is it ok that this verb handler doesn't invoke syncAllSensors?

Reply:
Yeah it seem we've missed to increment here

Reply:
I'd actually propose we change the Sensor API a bit and add function that increments and syncs values (currently it's easy to miss the sync call). Thoughts?

Reply:
There is no need to invoke `syncAllSensors` in the commit handler because it is taken care of here: https://github.com/datastax/cassandra/blob/cf895973ebf3628279c37abb58ab35ea8e527a84/src/java/org/apache/cassandra/db/SystemKeyspace.java#L699

There reason prepare and propose handler add it is because `INTERNODE_BYTES` is calculated _after_  `transferPaxosSensorBytes` is invoked. Commit handler has a NoPayload message so no extra internode bytes need to be added (and consequently no need to sync).

Few notes:
* Sergio made syncAllSensors idempotent so calling sync multiple times is safe (https://github.com/datastax/cassandra/pull/1119)
* Unit tests would fail if sync was required and not added bc we always tests registry sensors (https://github.com/datastax/cassandra/blob/abef30a8f03d592dd89a116495de53949f9f709a/test/unit/org/apache/cassandra/sensors/SensorsInternodeTest.java#L257)
* Auto sync was discussed (and dismissed) in light of performance improvement: https://github.com/datastax/cassandra/pull/1156